Makefile

```
@@ -127,3 +130,27 @@ lint_country_dialing_codes: update_pinpoint_supported_countries ## Checks that c
 
 check_asset_strings: ## Checks for strings
 	find ./app/javascript -name "*.js*" | xargs ./scripts/check-assets
+
+build_artifact: ## Builds zipped tar file artifact with IDP source code and Ruby/JS dependencies
+	@echo "Building artifact into $(ARTIFACT_DESTINATION_FILE)"
+	bundle config set --local cache_all true
+	bundle package
+	tar \
+	  --exclude './config/agencies.yml' \
+	  --exclude './config/iaa_gtcs.yml' \
+	  --exclude './config/iaa_orders.yml' \
+	  --exclude './config/iaa_statuses.yml' \
+	  --exclude './config/integration_statuses.yml' \
+	  --exclude './config/integrations.yml' \
+	  --exclude './config/partner_account_statuses.yml' \
+	  --exclude './config/partner_accounts.yml' \
+	  --exclude './config/service_providers.yml' \
+	  --exclude='./certs/sp' \
+	  --exclude='./identity-idp-config' \
+	  --exclude='./tmp' \
+	  --exclude='./node_modules' \
+	  --exclude='./geo_data/GeoLite2-City.mmdb' \
+	  --exclude='./pwned_passwords/pwned_passwords.txt' \
+	  --exclude='./vendor/ruby' \
+	  --exclude='./config/application.yml' \
+	  -cf - "." | "$(GZIP_COMMAND)" > "$(ARTIFACT_DESTINATION_FILE)"
```

More make magic now that the target is a file:
```suggestion
	  -cf - "." | "$(GZIP_COMMAND)" > $@
```